src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs

```diff
@@ -1012,6 +1023,35 @@ private void ShowUpdatePackages(IVsWindowFrame windowFrame, ShowUpdatePackageOpt
             packageManagerControl?.ShowUpdatePackages(updatePackageOptions);
         }
 
+        public async Task ShowReadMeMarkdownToolWindowAsync(string filePath)
+        {
+            await JoinableTaskFactory.SwitchToMainThreadAsync(DisposalToken);
+
+            var toolWindow = FindToolWindow(typeof(ReadMeMarkdownToolWindow), 0, create: true);
+
+            if (!(toolWindow is ReadMeMarkdownToolWindow markdownToolWindow))
+            {
+                return;
+            }
+
+            await markdownToolWindow.UpdateAsync(filePath);
+
+            Guid persistenceSlot = typeof(ReadMeMarkdownToolWindow).GUID;
+
+            IVsUIShell vsUIShell = (IVsUIShell)GetService(typeof(SVsUIShell));
+            if (vsUIShell != null)
+            {
+                IVsWindowFrame windowFrame;
+                ErrorHandler.ThrowOnFailure(vsUIShell.FindToolWindow(0, persistenceSlot, out windowFrame));
+                if (windowFrame != null)
+                {
+                    //show tool window
+                    ErrorHandler.ThrowOnFailure(windowFrame.Show());
```

Comment:
Some error handling here is missing.

Firstly, if either `vsUIShell` or `windowFrame` is null, literally nothing happens. The customer won't see the readme that they expect to see (assuming they know the package contains a readme and that they know it's supposed to be disaplayed automatically).  Similarly, we the nuget team will get no information to help us understand the errors. NuGet won't be working as per the design spec, and nobody will know.

Secondly, I followed the call chain to see where `ErrorHandler.ThrowOnFailure`'s exception is caught, which is in `NuGet.PackageManagement/NuGetPackageManager.cs`, and it's going to cause the package installation to be rolled back.  Is this the customer experience we want?